internal/server/v1/page/golden/place_page/zuid-nederland.sample.json

```diff
@@ -18133,10 +18133,41 @@
     }
   },
   "validCategories": {
-    "country/NLD": {},
-    "nuts/NL4": {},
-    "nuts/NL41": {},
-    "nuts/NL42": {}
+    "country/NLD": {
+      "category": [
```

Comment:
why were these empty on the LHS?

Reply:
They should contain the actual valid categories from the definition. To me, it's bug from previous implementation.